CareKit/CareKit/Lists/Controller/OCKDailyPageViewController.swift

```diff
@@ -110,6 +110,10 @@ UIPageViewControllerDataSource, UIPageViewControllerDelegate {
 
     // MARK: - Properties
 
+    open func selectDate(_ date: Date, animated: Bool) {
+        calendarWeekPageViewController.selectDate(date, animated: animated)
```

Comment:
```suggestion
        let previousDate = selectedDate
        guard !Calendar.current.isDate(previousDate, inSameDayAs: date) else { return }
        calendarWeekPageViewController.selectDate(date, animated: true)
        weekCalendarPageViewController(calendarWeekPageViewController, didSelectDate: date, previousDate: previousDate)
```

Reply:
Then the `pressedToday(sender:)` method on line 155 can be changed to just

```swift
@objc
private func pressedToday(sender: UIBarButtonItem) {
    selectDate(Date(), animated: true)
}
```

Reply:
Sounds Good !